templates/notification.yaml.j2

```diff
@@ -78,12 +78,12 @@ logging: # Used in logging.dictConfig
             backupCount: 5  # Keep 5 older logs around
     loggers:
         kazoo:
-            level: WARN
```

Comment:
These 3 loggers are there to suppress log messages from the libraries we use that are very verbose but aren't useful for our application. Outside of a development environment these should never been lower than WARN. Given that I think we could just leave them hardcoded at WARN or ERROR and only have the root log level configurable.